.github/workflows/checker-framework.yml

```diff
@@ -40,8 +40,9 @@ jobs:
         run: groovy ./.ci/checker-framework.groovy ${{ matrix.profile }} || true
 
       - name: Git Diff on fail
-        if: failure()
-        run: git diff > target/${{ matrix.profile }}.patch
+        # if: failure()
+        # run: git diff > target/${{ matrix.profile }}.patch
+        run: git diff
 
       - name: Share patch as artifact to apply on local
```

Comment:
Patch needs the same changes as well.

Reply:
I did this on purpose, but as you mention this also , ok, we print in output + in file and store as artifacts

Reply:
Why `git diff && git diff`?